wherehows-web/app/components/dataset-spinner.js

```diff
@@ -0,0 +1,13 @@
+import Component from '@ember/component';
+import { set } from '@ember/object';
+
+export default Component.extend({
```

Comment:
Take a look at the notifications module, the preferred approach would be to augment that to fit this use case rather than a stand alone spinner component